apps/server/src/shared/domain/domainobject/user.do.ts

```diff
@@ -35,6 +35,10 @@ export class UserDO extends BaseDO {
 
 	preferences?: Record<string, unknown>;
 
+	lastLoginSystemChange?: Date;
+
+	outdated?: Date;
+
 	constructor(domainObject: UserDO) {
```

Comment:
Why is a DO used to construct a DO?

Reply:
This is just older code, where we treated DOs as POJOs. This is of course subject to change under new decisions